tools/fuzzer/symbol_table_test.cc

```diff
@@ -296,3 +322,55 @@ TEST_F(PopulateSymbolTableTest, TaggedTypesAndFields) {
     expect_field(nested, "f1", ScalarType::SignedInt);
   }
 }
+
+TEST_F(PopulateSymbolTableTest, Enums) {
+  {
+    const EnumType type("CStyleEnum", false);
+    auto enum_it = symtab_.enums().find(type);
+    ASSERT_NE(enum_it, symtab_.enums().end());
+    const auto& literals = enum_it->second;
+
+    EXPECT_THAT(literals,
+                UnorderedElementsAre(EnumConstant(type, "CStyleEnum::VALUE1"),
+                                     EnumConstant(type, "CStyleEnum::VALUE2"),
+                                     EnumConstant(type, "CStyleEnum::VALUE3")));
+  }
+
+  {
+    const EnumType type("ns::CStyleEnum", false);
+    auto enum_it = symtab_.enums().find(type);
+    ASSERT_NE(enum_it, symtab_.enums().end());
+    const auto& literals = enum_it->second;
+
+    EXPECT_THAT(literals,
+                UnorderedElementsAre(EnumConstant(type, "ns::CStyleEnum::V1"),
+                                     EnumConstant(type, "ns::CStyleEnum::V2"),
+                                     EnumConstant(type, "ns::CStyleEnum::V3")));
+  }
+
+  {
+    const EnumType type("EnumClass", kHasScopedEnums);
+    auto enum_it = symtab_.enums().find(type);
+    ASSERT_NE(enum_it, symtab_.enums().end());
+    const auto& literals = enum_it->second;
+
+    EXPECT_THAT(literals,
+                UnorderedElementsAre(EnumConstant(type, "EnumClass::ZERO"),
+                                     EnumConstant(type, "EnumClass::ONE"),
+                                     EnumConstant(type, "EnumClass::TWO"),
+                                     EnumConstant(type, "EnumClass::THREE")));
+  }
+
+  {
+    const EnumType type("ns::EnumClass", kHasScopedEnums);
+    auto enum_it = symtab_.enums().find(type);
+    ASSERT_NE(enum_it, symtab_.enums().end());
+    const auto& literals = enum_it->second;
+
+    EXPECT_THAT(literals, UnorderedElementsAre(
+                              EnumConstant(type, "ns::EnumClass::ZERO"),
+                              EnumConstant(type, "ns::EnumClass::ONE"),
```

Comment:
Should `UnorderedElementsAre` be on the next line, same as in the code above? Weird that `clang-format` doesn't complain

Reply:
If it is the same as in the code above, then the `ns::EnumClass::THREE` line will be 82 characters long.

Reply:
Oh I see, didn't notice that